Userland/Games/Chess/ChessWidget.cpp

```diff
@@ -399,8 +398,8 @@ void ChessWidget::set_piece_set(StringView set)
 Optional<Chess::Square> ChessWidget::mouse_to_square(GUI::MouseEvent& event) const
 {
     int const min_size = min(width(), height());
-    int const widget_offset_x = (window()->width() - min_size) / 2;
-    int const widget_offset_y = (window()->height() - min_size) / 2;
+    int const widget_offset_x = 0;
+    int const widget_offset_y = 0;
```

Comment:
That doesn't seem right :thinking:
Can you elaborate a little bit on why you set that to zero?
And if it's the correct solution, please remove the variable instead of keeping useless noise.

Reply:
> Can you set your editor to use 72-characters line wrap instead of 52?

Ah, I misunderstood the requirement, I was thinking it was a _max_ of 72 so I was cutting them short to be safe. Will amend

> Can you elaborate a little bit on why you set that to zero?

Before, it was set to stay in the middle of the window. But when you add in a widget on the side, it tries to still stay in the middle of the window, resulting in part of the board being cut off, so I changed it so that it stays in the top left corner. Will clean it up if this approach is ok.

Reply:
I played a little bit with it locally, I get it now. This was used to center the widget, but this is not needed anymore.
This is a perfectly fine approach, except one detail: it still needs to be centered vertically!

Reply:
It's now vertically centered